include/openthread/platform/radio.h

```diff
@@ -260,6 +260,8 @@ typedef struct otRadioFrame
 
     uint8_t mRadioType; ///< Radio link type - should be ignored by radio driver.
 
+    uint8_t mIid; ///< Interface Id for the radio packet.
```

Comment:
We would need this to avoid modifying the signature of callback function(s) such as https://github.com/openthread/openthread/pull/8914/files/b2fa77ec0122dff82ea865303cbe689d26f1273a#diff-d714521b46e56369c7351ae0d9980cd53fe325ea448af3b590e44bf3f929c8ddR131

Reply:
Agree; I updated the documentation.